lib/features/add_property/view/widgets/add_features_view.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/data/property_data.dart';
+import 'package:hatspace/dimens/hs_dimens.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/strings/l10n.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+
+class AddFeaturesView extends StatefulWidget {
```

Comment:
StatefulWidget is not needed here

Reply:
@suesitran Currently i'm using ValueNotifier<List<Feature>>, ValueNotifer class has dispose method. So i used stateful to in order to dispose it

Reply:
Good point 👍